### PR TITLE
ech

### PR DESCRIPTION
**Pull request by NullDagaf**
## About The Pull Request

fixes linter errors

### CODE REVIEW

**Comment by projectkepler-ru** [comment]
thanks hun :3

File: modular_zubbers/code/game/objects/items/holy_weapons.dm [hunk 1]
```diff
@@ -3,6 +3,7 @@
 	desc = "A staff used by traditional bishops and popes."
 	icon = 'modular_zubbers/icons/obj/items_and_weapons.dmi'
 	icon_state = "papal_staff"
+	worn_icon_state = "nullrod"
```
Comment (Cyprex) [inline modular_zubbers/code/game/objects/items/holy_weapons.dm:6]: Wait, THAT's what was needed? ugh.

Comment (NullDagaf) [reply modular_zubbers/code/game/objects/items/holy_weapons.dm:6]: Yep.

**Comment by Cyprex** [comment]
Thanks a ton for this, it's most appreciated! ❤️

**Pull request merged**